ui/src/.eslintrc.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     ],
     plugins: ['flow-header', 'flowtype'],
     rules: {
+        'no-plusplus': 'off',
```

Comment:
I wouldn't turn in off in general, but rather inline it: `//eslint-disable-next-line no-plusplus` because in general we follow the airbnb styleguide.

Reply:
A fair suggestion!